packages/actor-query-operation-sparql-endpoint/test/ActorQueryOperationSparqlEndpoint-test.ts

```diff
@@ -158,7 +158,9 @@ describe('ActorQueryOperationSparqlEndpoint', () => {
           namedNode('http://o')) };
       const thisActor = new ActorQueryOperationSparqlEndpoint({ name: 'actor', bus, mediatorHttp: thisMediator });
       return expect(new Promise((resolve, reject) => {
-        thisActor.run(op).then(async (output) => output.bindingsStream.on('error', resolve));
+        thisActor.run(op).then(async (output) => output.bindingsStream.on('error', resolve)).catch((e) => {
+          reject(e);
```

Comment:
In cases like this, we likely want to use the second argument of the `then` function (because semantics are slightly different when you have a `catch` method after). So `.then(async (output) => output.bindingsStream.on('error', resolve), reject)`